### ISSUE TITLE
Some streams requires an account to be viewable

### ISSUE THREAD

**Issue by stemy2** [Free-TV/IPTV#427]
I tried all streams linked to a belgian TV and none of them seemed to work. They are still playable on their official site but only with an account. Is it the reason why the link mentioned in the m3u file is not working anymore ?

**Comment by KAMI911** [comment]
Channels that require an account/login are paid or registration-gated by definition, which is out of scope for this repo ("only free channels", no login walls). Not something to fix here.

**Comment by KAMI911** [comment]
Fixed most of it: 9 of the 13 Wallonia/Flanders channels sharing the streamhost.cc CDN were 404ing (RTL-Be, La Une, Club RTL, La Trois, Plug RTL, Canvas, VTM, VIER, VIJF) — marked invalid. Tipik on the same CDN still works, as do LN24, BX1 and Bruzz. EEN and RTS-type raw-IP hosts time out rather than error, which could be a Belgium-only geo-restriction rather than genuinely dead — left those as-is. Leaving open for replacements on the confirmed-dead ones.
